systemtest/aggregation_test.go

```diff
@@ -328,6 +328,51 @@ func TestServiceSummaryMetricsAggregation(t *testing.T) {
 	systemtest.ApproveEvents(t, t.Name(), result.Hits.Hits)
 }
 
+func TestServiceSummaryMetricsAggregationOverflow(t *testing.T) {
+	systemtest.CleanupElasticsearch(t)
+	srv := apmservertest.NewUnstartedServerTB(t)
+	srv.Config.Aggregation = &apmservertest.AggregationConfig{
+		ServiceTransactionMaxGroups: 2,
+	}
+	require.NoError(t, srv.Start())
+
+	f := func(env string) {
```

Comment:
Maybe we can rename this to `sendTransaction`? `f` seems unnecessarily short 😄

Reply:
it is unnecessarily short :facepalm: